TypeCobol/Compiler/Sql/CodeElements/Statements/TruncateStatement.cs

```diff
@@ -0,0 +1,40 @@
+﻿using TypeCobol.Compiler.CodeElements;
+using TypeCobol.Compiler.Sql.Model;
+
+namespace TypeCobol.Compiler.Sql.CodeElements.Statements
+{
+    /// <summary>
+    /// SQL Truncate Statement Code Element.
+    /// </summary>
+    public class TruncateStatement : SqlStatementElement
+    {
+        public enum StorageManagementOption
+        {
+            DropStorage,
+            ReuseStorage
+        }
+        public enum DeleteTriggersHandlingOption
+        {
+            IgnoreDeleteTriggers,
+            RestrictWhenDeleteTriggers
+        }
+        public TableViewCorrelationName TableName { get; }
+        public SyntaxProperty<StorageManagementOption> StorageManagement { get; }
+        public SyntaxProperty<DeleteTriggersHandlingOption> DeleteTriggersHandling { get; }
+        public SyntaxProperty<bool> IsImmediate { get; }
+
+        public TruncateStatement(TableViewCorrelationName tableName, SyntaxProperty<StorageManagementOption> storageManagement, SyntaxProperty<DeleteTriggersHandlingOption> deleteTriggersHandling, SyntaxProperty<bool> isImmediate) : base(CodeElementType.TruncateStatement, StatementType.TruncateStatement)
+        {
+            this.TableName = tableName;
+            this.StorageManagement = storageManagement;
+            this.DeleteTriggersHandling = deleteTriggersHandling;
+            IsImmediate = isImmediate;
```

Comment:
`this.` is optional (see ReSharper notice about it). I don't mind if you use it however why not apply the same style for all properties ?